site/docs/pages/swap/swap.mdx

```diff
@@ -101,6 +101,12 @@ export default function SwapComponents() {
   <SwapComponents />
 </App>
 
+:::danger
```

Comment:
Should we use `warning` instead? As we are alerting people instead of giving them a message they did something wrong.

Reply:
we use `warning` for a different message above, we want to highlight this for users to understand the `Swap` is not broken but intentionally disabled

Reply:
awesome